certbot/src/certbot/_internal/cli/cli_utils.py

```diff
@@ -131,6 +131,24 @@ def add_domains(args_or_config: Union[argparse.Namespace, configuration.Namespac
     return validated_domains
 
 
+class _IPAddressAction(argparse.Action):
+    """Action class for parsing IP addresses."""
+
+    def __call__(self, parser: argparse.ArgumentParser, namespace: argparse.Namespace,
+                 values: Union[str, Sequence[Any], None],
```

Comment:
```suggestion
                 values: str | Sequence[Any] | None,
```
nit: new-style is nicer